src/kfs_frontend/kfs_grpc_inference_service.cpp

```diff
@@ -41,7 +46,10 @@
 #include "../tensorinfo.hpp"
 #include "../timer.hpp"
 #include "../version.hpp"
-
+#include "mediapipe/framework/calculator_graph.h"
+#include "mediapipe/framework/port/logging.h"
```

Comment:
Do we need those includes? Shouldnt those be hidden inside mediapipe_internal/ cpp units?